### DISCUSSION TITLE
FastAPI logo copyright - Use of fastapi logo in graphics etc.

### DISCUSSION THREAD

**Discussion by mokrueger** [fastapi/fastapi#3646] [Questions]
Hello there,

I was curious to how the logo of fastapi is licensed / under copyright.
If i'd use the fastapi logo in a server architecture graphic (e.g. in a thesis) in what ways would i need to give credits in terms of copyright etc.

@tiangolo

**Comment by ghandic** [comment]
Given that it is in the repository of an MIT licensed project, it would suggest that the logo is also MIT licensed. However, I am no lawyer and it would be at the discretion of @tiangolo
